src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -167,6 +168,10 @@ def save_table(self, full_name: str, rows: Sequence[DataclassInstance], klass: D
         df = self._spark.createDataFrame(rows, self._schema_for(klass))
         df.write.saveAsTable(full_name, mode=mode)
 
+    def get_table_size(self, table_full_name: str) -> int:
```

Comment:
doesn't belong here. move to `TableSizeCrawler`.

Reply:
Table size crawler doesn't have spark context.

Reply:
You get it from spark session you create there.

This method doesn't belong here. There's no way to get it from SQL, so "no" means "no"

Reply:
The spark session is private. Should I make it public?

Reply:
Fixed

Reply:
Revert this change from here

Reply:
Fixed